docs/source/guides/libraries.rst

```diff
@@ -67,12 +67,256 @@ Python library
 Where to get
 ^^^^^^^^^^^^
 
+Prerequirements:
+
+- cmake(3.11 or higher)
+- git
+- g++
+- boost(1.65 or higher, only system and filesystem)
+- swig(3.0.12 can be built --without-pcre)
+- protobuf
+
+Install iroha python libraries:
+
+- Through pip
+
+    .. code:: sh
+
+      pip install iroha
+
+    For the latest version
+
+    .. code:: sh
+
+      pip install -i https://testpypi.python.org/pypi iroha
+
+- Source code
+
+    .. code:: sh
+
+      git clone https://github.com/hyperledger/iroha.git
+      cd iroha
+
+    For the latest version checkout to develop branch
+
+    .. code:: sh
+
+      git checkout develop
+
+    .. code:: sh
+
+      cmake -H. -Bbuild -DSWIG_PYTHON=ON -DSHARED_MODEL_DISABLE_COMPATIBILITY=ON -DSUPPORT_PYTHON2=ON;
+      cmake --build build -- -j4
+
+    After this you can find iroha python library in **iroha/build/shared_model/bindings** folder.
+
+Install iroha protobuf files:
+
+  First of all you need to clone iroha repository
+
+  .. code:: sh
+
+      pip install grpcio_tools
+      git clone https://github.com/hyperledger/iroha.git
+      cd iroha
+      protoc --proto_path=schema --python_out=build block.proto primitive.proto commands.proto queries.proto responses.proto endpoint.proto
+      python -m grpc_tools.protoc --proto_path=schema --python_out=build --grpc_python_out=build endpoint.proto yac.proto ordering.proto loader.proto
+
+  Protobuf files can be found in **iroha/build** folder ('\*_pb2\*.py' files)
+
 How to use/import
 ^^^^^^^^^^^^^^^^^
 
+In order to specify iroha libraries location:
+
+.. code:: sh
+
+  import sys
+  sys.path.insert(0, 'path/to/iroha/libs')
+
+
+Import iroha and all of the protobuf modules that you need:
+
+.. code:: sh
+
+  import iroha
+  import block_pb2
+  import endpoint_pb2
+  import endpoint_pb2_grpc
+  import queries_pb2
+
 Example code
 ^^^^^^^^^^^^
 
+Note: work with raw data can be different in Python 2 and Python 3.
```

Comment:
This should be styled as a block from here: https://sphinx-rtd-theme.readthedocs.io/en/latest/demo/demo.html#admonitions please check the raw code on GitHub